Fixes the following CMake error:
CMake Error at CMakeLists.txt:17 (target_link_libraries):
  The plain signature for target_link_libraries has already been used with
  the target "RmlCore".  All uses of target_link_libraries with a target must
  be either all-keyword or all-plain.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -563,13 +563,13 @@
 if(NOT BUILD_FRAMEWORK)
 	target_include_directories(RmlCore PRIVATE ${CORE_INCLUDE_DIRS})
 	target_include_directories(RmlCore INTERFACE $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/Include> $<INSTALL_INTERFACE:include>)
-	target_link_libraries(RmlCore ${CORE_LINK_LIBS})
+	target_link_libraries(RmlCore PRIVATE ${CORE_LINK_LIBS})
 	target_link_libraries(RmlDebugger RmlCore)
 	target_compile_definitions(RmlCore PRIVATE ${CORE_PRIVATE_DEFS})
 	target_compile_definitions(RmlCore PUBLIC ${CORE_PUBLIC_DEFS})
 else()
 	target_include_directories(RmlUi PRIVATE ${CORE_INCLUDE_DIRS})
-	target_link_libraries(RmlUi ${CORE_LINK_LIBS})
+	target_link_libraries(RmlUi PRIVATE ${CORE_LINK_LIBS})
 	target_compile_definitions(RmlUi PRIVATE ${CORE_PRIVATE_DEFS})
 	target_compile_definitions(RmlUi PUBLIC ${CORE_PUBLIC_DEFS})
 endif()
